practice-one/src/assets/css/global.css

```diff
@@ -6,9 +6,13 @@
   --primary-font: "Abel";
   --secondary-font: "Montserrat";
   --black-color: #000;
+  --white-color: #fff;
   --gray-color: #7d7d7d;
   --brown-color: #ebcb80;
   --orange-color: #e8b229;
+  --fz-primary: 18px;
```

Comment:
Should split and add a comment for each part as font, color,...

Reply:
I fix it 
![image](https://github.com/HoangLiarPhuongLe/html_css_training/assets/110210274/6a677859-fd4b-4649-96be-912dad5a143f)